articles/sentinel/detect-threats-custom.md

```diff
@@ -164,7 +164,7 @@ If you see that your query would trigger too many or too frequent alerts, you ca
 
     | Field name | Contains | Running the query in this field<br>results in... |
     | - | :-: | :-: |
-    | **Query** | The compressed record of the event that generated this instance of the alert | The event that generated this instance of the alert |
+    | **Query** | The compressed record of the event that generated this instance of the alert | The event that generated this instance of the alert limeted to 10240 bytes  |
```

Comment:
Really? 10K? Isn't that small?
Also, small spelling correction, and I added a line break:
```suggestion
    | **Query** | The compressed record of the event that generated this instance of the alert | The event that generated this instance of the alert;<br>limited to 10240 bytes  |
```